spec/presenters/scope_filters_presenter_spec.rb

```diff
@@ -50,90 +55,126 @@
         "/",
         "http://gov.uk"
       ].each do |root_style_path|
-        presenter = described_class.new(path: root_style_path)
-        expect(presenter.path).to eq('/')
+        presenter = described_class.new(paths: [root_style_path])
+        expect(presenter.paths).to eq('/')
       end
     end
 
-    it "is nil for blank URLs" do
+    it "is root path for blank URLs" do
       [
         nil,
         ""
       ].each do |blank_url|
-        presenter = described_class.new(path: blank_url)
-        expect(presenter.path).to be_nil
+        presenter = described_class.new(paths: [blank_url])
+        expect(presenter.paths).to eq "/"
       end
     end
+
+    it "is root path for multiple blank URLs" do
+      presenter = described_class.new(paths: [nil, ''])
+      expect(presenter.paths).to eq '/'
+    end
+
+    it "is root path for missing URLs in a list" do
+      missing_urls = [' ', ' ', 'http://www.gov.uk/some-path']
+
+      presenter = described_class.new(paths: missing_urls)
+      expect(presenter.paths).to eq '/, /some-path'
+    end
   end
 
-  describe "#filtered?" do
-    it 'is false when path and organisation are blank' do
-      presenter = described_class.new(path: nil, organisation_slug: nil)
+  describe '#filtered?' do
+    it 'is false when `paths` and `organisation_slug` are blank' do
+      presenter = described_class.new(paths: nil, organisation_slug: nil)
       expect(presenter).not_to be_filtered
     end
 
-    it 'is true when path is provided and organisation is blank' do
-      presenter = described_class.new(path: '/done/buying-a-new-hat', organisation_slug: nil)
+    it 'is true when `paths` is provided and `organisation_slug` is blank' do
+      presenter = described_class.new(paths: ['/done/buying-a-new-hat'], organisation_slug: nil)
       expect(presenter).to be_filtered
     end
 
-    it 'is true when organisation is provided and path is blank' do
-      presenter = described_class.new(path: nil, organisation_slug: 'department-of-hats')
+    it 'is true when `organisation_slug` is provided and `paths` is blank' do
+      presenter = described_class.new(paths: nil, organisation_slug: 'department-of-hats')
       expect(presenter).to be_filtered
     end
 
-    it 'is true when both organisation and path are provided' do
-      presenter = described_class.new(path: '/done/buying-a-new-hat', organisation_slug: 'department-of-hats')
+    it 'is true when both `organisation_slug` and `path` are provided' do
+      presenter = described_class.new(paths: ['/done/buying-a-new-hat'], organisation_slug: 'department-of-hats')
       expect(presenter).to be_filtered
     end
   end
 
-  describe "#invalid_filter?" do
-    it 'is true when path and organisation are blank' do
-      presenter = described_class.new(path: nil, organisation_slug: nil)
+  describe '#invalid_filter?' do
+    it 'is true when path`` and `organisation_slug` are blank' do
```

Comment:
Minor: the backticks here should be around `path`

Reply:
🤦‍♀️ Corrected